lib/docblockr-worker.js

```diff
@@ -296,6 +311,27 @@ module.exports =
       this.write(editor, snippet);
     };
 
+    DocBlockrAtom.prototype.parse_python_command = function(inline) {
+      var editor = this.get_relevant_editor();
+      if (typeof editor === 'undefined' || editor === null) {
```

Comment:
Since editor can't ever be `false`. I think checking `!editor` is enough here.

Reply:
I just took the syntax from parse_command. I'm not really use to atom plug-in dev but i will fix this.